packages/web-vite/src/pages/LibraryPage.tsx

```
@@ -663,19 +721,25 @@ const LibraryPage: React.FC = () => {
               All
             </button>
             <button
-              className={`folder-tab ${activeFolder === 'inbox' ? 'active' : ''}`}
+              className={`folder-tab ${
+                activeFolder === 'inbox' ? 'active' : ''
+              }`}
               onClick={() => setActiveFolder('inbox')}
             >
               Inbox
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)